### DISCUSSION TITLE
Chapter endnotes fail to display on mac desktop app

### DISCUSSION THREAD

**Discussion by dm5n** [edrlab/thorium-reader#2639] [Ideas]
Hello, the Notes section at the end of each chapter fail to display on Thorium 3 on macOS 15.0.1

The Notes section displays properly on my Android epub reader (Librera).

![Screenshot 2024-10-06 at 22 43 57](https://github.com/user-attachments/assets/346a0dc6-0393-4cfd-a08b-38629fefd8f4)

**Comment by danielweck** [comment]
hello, please try disabling the "popup footnotes" in Thorium's configuration.

**Comment by dm5n** [comment]
Yes, that fixed it. 

The setting was 'Disable footnotes' in the individual book's settings under the 'Aa' icon.

Perhaps this setting could be better worded 'Popup footnotes' or 'Enable inline footnotes' or similar?

In any case, Thankyou :)

**Comment by danielweck** [comment]
This is a tricky UX problem ... personally I have never been a fan of automatically hiding sections marked with the footnotes/endnotes role (especially as this only applies to aside, not to other HTML containers). But I know there are users who expect this behaviour when the EPUB "popup footnotes" feature is activated.
I am converting this issue to a discussion...
